src/Whoops/Util/TemplateHelper.php

```diff
@@ -22,6 +22,16 @@ class TemplateHelper
     private $variables = array();
 
     /**
+     * @var HtmlDumper
+     */
+    private $htmlDumper;
```

Comment:
hmm usage of the helper on different pages is actually a good argument. I guess having the properties will also ease unit testing because we dont depend on a semi-global state.

agree, leave it as is.